### PR TITLE
Feature/FPV-18 UI servers upload/save images

### PR DESCRIPTION
**Pull request by kendrick-807**
Implemented candidate/create route so that the image will be saved to amazon s3, whereas the image URL, name, age and message will be saved to mongoDB.

### CODE REVIEW

File: routes/candidateRoutes.js [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
 const candidateRouter = express.Router();
+const multer = require('multer');
+const dotenv = require('dotenv');
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3');
 const Candidate = require('../models/Candidate');
 const { authorizedOrdinaryUser, authorizedAdmin } = require('../utilities/utilities');
 const logger = require('../utilities/logger');
 
+const storage = multer.memoryStorage();
+const upload = multer({ storage });
```
Comment (phamduylong) [inline routes/candidateRoutes.js:11]: So, storing on both Amazon S3 and server's physical storage? What is Amazon S3 for then?

Comment (kendrick-807) [reply routes/candidateRoutes.js:11]: It's just a temporary storage and let me access to the file data

Comment (kendrick-807) [reply routes/candidateRoutes.js:11]: 螢幕截圖 2023-10-29 下午12 01 17

Comment (phamduylong) [reply routes/candidateRoutes.js:11]: Aha. Good then.

**Review by phamduylong — APPROVED** [review]

**Pull request merged**